### DISCUSSION TITLE
Feature Request [Amigo]: Audio Taps when you tap touchscreen

### DISCUSSION THREAD

**Discussion by odudex** [selfcustody/krux#263] [Ideas]
By [HugoEnder](https://github.com/hugoender)

Would be awesome to have the speaker output a tap sound (similar to the tap sound an iPhone makes when you press keys on a keyboard) whenever you successfully touch a menu item on the screen or when you tap to move on to next page on pages with no menu items.

Original: https://github.com/odudex/krux/issues/8

Additional info: I managed to play audio on Amigo, but did not implement the feature yet.

**Comment by tadeubas** [comment]
I think one of the problems is the need to copy the file to flash or SD card to be able to play it.

**Comment by 2oh1** [comment]
Hopefully any audio would be an optional feature.

Speaking of audio/visual cues, one thing I really like about entering text on Krux is the way the screen flashes a tiny bit in recognition that a letter was pressed.  That tiny visual cue is really helpful, especially when entering smaller characters, and especially when entering a space between words.  For around $50, the Amigo is obviously a cheap device, but the more I use it for this purpose, the more I love it.  And the more I use it, the more I notice the attention to detail you folks are putting into Krux.  Kudos!
